### ISSUE TITLE
Game crashes adding The Vault mod to my modpack

### ISSUE THREAD

**Issue by YaBoiChips** [Iskallia/Vault-public#813]
[latest.log](https://github.com/Iskallia/Vault-public-S1/files/8076328/latest.log)

whenever i launch the game i get this error 

![image](https://user-images.githubusercontent.com/63213388/154187070-fdae2ad8-1b2b-4512-834d-e2eb08e1c610.png)

it seems to be a mixin issue.

[modlist.txt](https://github.com/Iskallia/Vault-public-S1/files/8076365/modlist.txt)

**Comment by Pancakezombie2** [comment]
The reason it is having issues more then likely, is because the vault mod is built around the specific mods in the Vault Hunters mod pack. So if you don't have those mods the mod can't function correctly.

**Comment by YaBoiChips** [comment]
> 

it showed that it needed dank storage and mekanism, which i added and it still didnt work

and if that is true they should add a list of mods required for pack makers as the log doesnt show it

**Comment by YokoSilver** [comment]
Yeah I second that. Cause I'm getting the same issues as well not knowing what mods are exactly needed in order to run this properly.

**Comment by X-Niter** [comment]
Digging a bit here, I found integration with // Jei, Curios, DankStorage, XeroxMiniMap, Mekanism

None other have been found. Hope it helps to any degree
![image](https://user-images.githubusercontent.com/30506033/164993577-ffebd654-1323-4fd7-8dfd-b2ea2f846138.png)

**Comment by X-Niter** [comment]
Really should add CurseForge dependency and maybe dependency checking that reports in the logs too. Or this will keep coming back over and over again haha.

**Comment by YokoSilver** [comment]
agreed. Want to download this mod? Sure here's the mod and all the other things it needs in order to run. Would be nice if that were the case. Makes ya wonder why make a mod that requires a huge amount of dependencies and not list that anywhere at all.
